src/main/java/Duke/Storage.java

```diff
@@ -0,0 +1,66 @@
+package Duke;
+
+import Duke.Tasks.Deadline;
+import Duke.Tasks.Event;
+import Duke.Tasks.Task;
+import Duke.Tasks.ToDo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.Scanner;
+
+public class Storage {
+
+    private String filePath;
+    private File f;
```

Comment:
I think the name f could be renamed to something more meaningful, e.g. file, although it's quite obvious here.
Usually use scratch variables for cases like temporary storage or loop indices.